juju/juju-gui
# Don't show ghost inspector deploy button with the deployer bar

This branch hides the ghost inspector deploy controls when the deployer bar is visible.

There is also drive-by to remove some CSS and a template that are no longer being used.

QA:
With no flags:
- Drag a service to the canvas.
- Check that the 'Cancel' and 'Deploy' buttons **are** visible in the inspector.

With mv flag:
- Drag a service to the canvas.
- Click on the service (the inspector should open).
- Check that the 'Cancel' and 'Deploy' buttons are **not** visible.
